index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Hien's portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <a href="http://" target="_blank" rel="noopener noreferrer"><img src="/image/logoipsum-265.svg" alt="Logo" class="logo"></a>
+        <nav >
+            <div class="hamberger"></div>
```

Comment:
[Optional] In order to match the design as closely as possible, kindly modify the hamburger icon to have equal widths and same color as the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/204986979-a306e098-9ff9-48fd-9869-85d0f4702533.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/204987038-43c02433-0d10-40d6-8760-2aec26ccf834.png)